packages/box/lib/utils/unbox.ts

```diff
@@ -9,7 +9,14 @@ import inquirer from "inquirer";
 import { boxConfig, unboxOptions } from "typings";
 import { promisify } from "util";
 
-async function verifyURL(url: string) {
+function verifyLocalPath(url: string) {
+  const configPath = path.join(url, "truffle-box.json");
```

Comment:
We probably should name this variable `path` and not `url`. What do you think?

Reply:
yes, it was my first intention. However it clashes with Node's `path` declared already in a upper scope (and used in this very same function on line 13)

Reply:
Ah yes! How about `targetPath`?

Reply:
probably `sourcePath` as the target can be confused with destination?